.github/workflows/server-release.yml

```diff
@@ -101,24 +186,58 @@ jobs:
           cache-to: type=gha,mode=max
           build-args: |
             TUIST_HOSTED=0
-            TUIST_VERSION=${{ steps.version.outputs.version }}
+            TUIST_VERSION=${{ needs.setup.outputs.next-version }}
             MIX_ENV=prod
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           github-token: ${{ secrets.TUIST_RELEASE_GITHUB_TOKEN }}
-      - uses: stefanzweifel/git-auto-commit-action@v5
-        id: commit
+
+  update-main:
+    name: Update main with released server
```

Comment:
as i mentioned in the previous review, I don't think we want the split to multiple jobs. We only need that for the iOS and macOS release since want to run the macOS and iOS build in parallel.